avm/res/document-db/database-account/mongodb-database/main.bicep

```diff
@@ -0,0 +1,56 @@
+metadata name = 'DocumentDB Database Account MongoDB Databases'
+metadata description = 'This module deploys a MongoDB Database within a CosmosDB Account.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent Cosmos DB database account. Required if the template is used in a standalone deployment.')
+param databaseAccountName string
+
+@description('Required. Name of the mongodb database.')
+param name string
+
+@description('Optional. Name of the mongodb database.')
+param throughput int = 400
+
+@description('Optional. Collections in the mongodb database.')
+param collections array = []
+
+@description('Optional. Tags of the resource.')
+param tags object?
+
+resource databaseAccount 'Microsoft.DocumentDB/databaseAccounts@2023-04-15' existing = {
+  name: databaseAccountName
+}
+
+resource mongodbDatabase 'Microsoft.DocumentDB/databaseAccounts/mongodbDatabases@2023-04-15' = {
+  name: name
+  parent: databaseAccount
+  tags: tags
+  properties: {
+    resource: {
+      id: name
+    }
+    options: contains(databaseAccount.properties.capabilities, { name: 'EnableServerless' }) ? null : {
+      throughput: throughput
+    }
+  }
+}
+
+module mongodbDatabase_collections 'collection/main.bicep' = [for collection in collections: {
+  name: '${uniqueString(deployment().name, mongodbDatabase.name)}-collection-${collection.name}'
+  params: {
```

Comment:
I may be mistaken, but I think this is not passing through all parameters

Reply:
You were right, fixed: https://github.com/Azure/bicep-registry-modules/pull/667/commits/27f7c366f2abaa7dbc7d053c6b2a9c1fbc7b6f45